app/models/mixins/relationship_mixin.rb

```diff
@@ -690,7 +690,9 @@ def remove_children(*child_objs)
     child_objs = child_objs.flatten.compact
     return child_objs if child_objs.empty?
 
-    return child_objs.each { |child| child.with_relationship_type(relationship_type) { child.update!(:parent => nil) } } if use_ancestry?
+    if use_ancestry?
+      return child_objs.each { |child| child.with_relationship_type(relationship_type) { child.update(:ancestor_ids => nil) } }
```

Comment:
this line is going to get me in trouble tomorrow but I can't get anything else to work.